synapse/storage/databases/main/media_repository.py

```diff
@@ -946,3 +972,42 @@ def _delete_url_cache_media_txn(txn: LoggingTransaction) -> None:
         await self.db_pool.runInteraction(
             "delete_url_cache_media", _delete_url_cache_media_txn
         )
+
+    async def get_is_hash_quarantined(self, sha256: str) -> bool:
+        """Get the metadata for a local piece of media
+
+        Returns:
+            None if the media_id doesn't exist.
+        """
+
+        def get_matching_media_txn(
+            txn: LoggingTransaction, table: str, sha256: str
+        ) -> bool:
+            # Return on first match
+            sql = f"""
+            SELECT sha256
```

Comment:
```suggestion
            SELECT 1
```

not that it matters, but since it's unused /shrug